mmv1/third_party/terraform/website/docs/ephemeral-resources/service_account_jwt.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+sidebar_current: "docs-google-service-account-jwt"
```

Comment:
These can be removed- they're not needed in frontmatter of markdown docs. They shouldn't be in the file they were copied from, and definitely not here!